### PR TITLE
add mastodon field to people and groups

### PR DESCRIPTION
**Pull request by Seunadex**
#### Is this solving an issue already in our backlog? If so, please mention which one
Yes https://github.com/rubycorns/rorganize.it/issues/656

#### Special things about this PR to be considered
We need to run migration

#### Do you have any open Questions you need to get answered?
nil

#### Please outline the areas that should be tested, if there are any:
1. Updating a person
2. Creating and updating a group

### CODE REVIEW

**Comment by Seunadex** [comment]
@lislis @sareg0 @vsmart ☝️Reviews, please!

File: app/views/people/_form.html.haml [hunk 1]
```diff
@@ -12,6 +12,9 @@
     .form-group
       = f.label(:twitter, "Your twitter handle")
       = f.text_field :twitter, placeholder: '@twitterhandle', class: 'form-control'
+    .form-group
+      = f.label(:mastodon, "Your mastodon handle")
+      = f.text_field :mastodon, placeholder: '@mastodon_handle', class: 'form-control'
```
Comment (lislis) [inline app/views/people/_form.html.haml]: I'd suggest the placeholder to be `username@domain.tld` or something like that. We need the domain and the user name

File: app/views/groups/_form.html.haml [hunk 2]
```diff
@@ -39,6 +39,9 @@
     .form-group
       = f.label(:twitter, "Twitter handle")
       = f.text_field :twitter, placeholder: '@twitterhandle', class: 'form-control'
+    .form-group
+      = f.label(:mastodon, "Mastodon handle")
+      = f.text_field :mastodon, placeholder: '@mastodon_handle', class: 'form-control'
```
Comment (lislis) [inline app/views/groups/_form.html.haml]: Also different placeholder `username@domain.tld`

**Comment by Seunadex** [comment]
@lislis feedback implemented

**Comment by Seunadex** [comment]
@lislis feedback implemented

**Comment by lislis** [comment]
(Sorry for the weird delay, GH problems...)
Thank you very much!
high five

**Pull request merged**